src/client/_clientwindow.py

```diff
@@ -767,6 +768,8 @@ def initMenus(self):
         self.actionSetSoundEffects.triggered.connect(self.updateOptions)
         self.actionSetOpenGames.triggered.connect(self.updateOptions)
         self.actionSetJoinsParts.triggered.connect(self.updateOptions)
+        self.actionSetNewbiesChannel.triggered.connect(self.updateOptions)
+        self.actionSetAutoJoinChannels.triggered.connect(lambda: self.on_actionSetAutoJoinChannels()) #zawa
```

Comment:
No need to use lambda here, just drop the parentheses. Connecting lambdas to signals is in general a bad idea, since you can never disconnect them.

Reply:
done.